core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
   Ce droit est utile quand vous ne voulez pas que les utilisateurs non authentifiés puissent voir les pages Jenkins \
   — retirez ce droit à l''utilisateur anonymous, puis \
   ajoutez le pseudo-utilisateur "authenticated" et accordez-lui le droit en lecture.
+Hudson.RunScriptsPermission.Description=\
+  Déprécié - veuillez utiliser le droit Overall/Administer à la place
+Hudson.NodeDescription=le nœud principal du contrôleur Jenkins
 
 Item.Permissions.Title=Job
-
+Item.CREATE.description=Crée un nouveau job.
+Item.DELETE.description=Supprime un job.
+Item.CONFIGURE.description=Modifie la configuration d''un job.
+Item.READ.description=Lire un job. (Vous pouvez interdire ce droit, mais autoriser Découvrir pour forcer un utilisateur anonyme à s''identifier et voir le job.)
+Item.RENAME.description=Renommer un job.
+ItemGroupMixIn.may_not_copy_as_it_contains_secrets_and_=Copie impossible de {0} car il contient des secrets et {1} possède {2}/{3} mais pas /{4}
```

```suggestion
ItemGroupMixIn.may_not_copy_as_it_contains_secrets_and_=Copie impossible de {0} car il contient des secrets et {1} possède {2}/{3} mais pas {4}
```
Not sure about that one.

---

The default properties has `/{4}`: https://github.com/jgreffe/jenkins/blob/2fc718da2a377d9dc411ebae95badf2f42212bef/core/src/main/resources/hudson/model/Messages.properties#L180

---

Indeed, strange... I lack the context of this message to understand why it's that way, can anyone share it?

---

Seems like paths:
https://github.com/jgreffe/jenkins/blob/2e9fff21ca8d5ed6b1def16791c381d36f074596/core/src/main/java/hudson/model/ItemGroupMixIn.java#L229-L235